CMakeLists.txt

```diff
@@ -1,9 +1,44 @@
-PROJECT(wingetopt)
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.0)
+
+project(getopt VERSION 1.0.0)
 
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/src)
-add_library(wingetopt STATIC src/getopt.c src/getopt.h)
+add_library(getopt STATIC src/getopt.c src/getopt.h)
```

Comment:
Why `STATIC`? Is it designed to not be used as shared? See:
* http://cgold.readthedocs.io/en/latest/tutorials/libraries/static-shared.html